### ISSUE TITLE
What is the novelty of this?

### ISSUE THREAD

**Issue by poke222444** [cjscholl/KNN_Text#1]
And how is this something new? I am really uncertain on what you are trying to achieve some of the code is all over the place have a look at this:

def tfidf_knn():
    """Method for determining nearest neighbors using bag-of-words model and K-Nearest Neighbor algorithm"""

    training_data = util.get_parser_data("amazon.csv")
    X_train, X_test, y_train, y_test = train_test_split(training_data["sentence"], training_data["sentiment"],
                                                        test_size=0.2, random_state=5)
    X_train, X_test = vectorize.createTFIDF(X_train, X_test, remove_stopwords=True, lemmatize=True, stemmer=False)
    knn = neighbors.KNeighborsClassifier(n_neighbors=5, weights='distance', algorithm='brute', leaf_size=30, p=2,
                                         metric='cosine', metric_params=None, n_jobs=1)

    knn.fit(X_train, y_train)
    predicted = knn.predict(X_test)
    acc = metrics.accuracy_score(y_test, predicted)
    print('KNN with TFIDF accuracy = ' + str(acc * 100) + '%')

??????
